docs/cli.rst

```diff
@@ -215,6 +217,55 @@ throughput (TP) and thereby, the amount of clock cycles the PIs need to be stall
 
 A ``network`` can also be simulated for comparison by using ``simulate -n``.
 
+
+Physical Simulation of SiDBs
+----------------------------
+
+Performing physical simulations on SiDB layouts is crucial for understanding layout behavior and
+facilitating rapid prototyping, eliminating the need for expensive and time-intensive fabrication processes.
+The command ``read --sqd`` (or ``read -s``) is used to import a SiDB layout from an sqd-file, a format compatible with `SiQAD <https://github.com/siqad/siqad>`_.
+The SiDB layout can be visualized using the ``print -c`` command. Currently, *fiction* provides two electrostatic physical simulators:
+an exact one (*QuickExact*) and a scalable one (*QuickSim*).
+
+QuickExact (``quickexact``)
+###########################
+
+*QuickExact* serves as an exact simulator, meticulously determining all physically valid charge distributions.
+It enumerates all possible charge distributions. However, by incorporating three techniques, namely
+1.) Physically-informed Search Space Pruning, 2.) Partial Solution Caching, and 3.) Effective State Enumeration, it provides
+a significant performance advantage of more than three orders of magnitude over ExGS from SiQAD. For additional details,
+see the `paper <https://www.cda.cit.tum.de/files/eda/2024_aspdac_efficient_exact_simulation.pdf>`_.
+
+Most important Parameters:
+
+- Relative permittivity (``-e``).
+- Thomas-Fermi screening (``-l``).
+- Energy transition level (0/-) (``-m``).
```

Comment:
Let's provide the variable names here as well, i.e. µ_ etc.

Reply:
@marcelwa I tried, but I did not manage to write epsilon_r etc. here. Do you have any idea what syntax I can use here?

Reply:
![Screenshot_20231211-130700.jpg](https://github.com/cda-tum/fiction/assets/19913620/38c7f275-8156-4091-a52e-eb49d8f767b0)

Reply:
@marcelwa What do I wrong?

Reply:
Hmm I don't know for sure, but I think there cannot be a space between the colon and the backtick